config/default.json

```diff
@@ -18,15 +18,9 @@
 	"ACTIVATION_LINK_PERIOD_OF_VALIDITY_SECONDS": 7200,
 	"ADMIN_TOGGLE_STUDENT_VISIBILITY": "enabled",
 	"LERNSTORE_MODE": "DISABLED",
+	"FEATURE_PROMETHEUS_ENABLED": true,
 	"PROMETHEUS": {
-		"METRICS_PATH": "/metrics",
-		"DURATION_BUCKETS_SECONDS": [0.01, 0.05, 0.1, 0.2, 0.5, 0.75, 1, 3, 5, 8, 15, 30, 60, 120]
-	},
-	"MATRIX_MESSENGER": {
```

Comment:
Is that on purpose that the matrix conf is deleted here?

Reply:
thanks!